stable/unifi/templates/captive-ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if (and .Values.captivePortalService.ingress.enabled (not .Values.unifiedService.enabled)) }}
+{{- $fullName := include "unifi.fullname" . -}}
+{{- $ingressPath := .Values.captivePortalService.ingress.path -}}
+{{- $unifiedServiceEnabled := .Values.unifiedService.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
```

Comment:
To avoid naming collisions with the primary unifi ingress, this should be changed to something like:

```yaml
name: {{ $fullName }}-captive
```